pkg/karmadactl/cmdinit/kubernetes/deploy.go

```diff
@@ -395,7 +395,7 @@ func (i *CommandInitOption) initKarmadaComponent() error {
 	// Create karmada-webhook
 	// https://github.com/karmada-io/karmada/blob/master/artifacts/deploy/karmada-webhook.yaml
 	klog.Info("create karmada webhook Deployment")
-	if err := i.CreateService(i.karmadaWebhookService()); err != nil {
+	if err := util.CreateOrUpdateService(i.KubeClientSet, i.karmadaWebhookService()); err != nil {
```

Comment:
```go
func CreateOrUpdateSecret(client kubeclient.Interface, secret *corev1.Secret) error {
	if _, err := client.CoreV1().Secrets(secret.Namespace).Create(context.TODO(), secret, metav1.CreateOptions{}); err != nil {
		if !apierrors.IsAlreadyExists(err) {
			return fmt.Errorf("unable to create service: %v", err)
		}
```
This function's error message is incorrect(`secret`, not `service`), could you correct this mistake?

Reply:
fixed